docs/guidelines.md

```diff
@@ -54,3 +54,7 @@ class MyApp extends React.Component {
   }
 }
 ```
+
+## Building Sass Files
+
+If you use the Sass files instead of the built CSS files found in the `/dist` directory, we recommend you use a package like [Autoprefixer](https://github.com/postcss/autoprefixer). Autoprefixer adds vendor prefixes to your CSS Output (ie. -moz-xyz, -webkit-xyz, etc). We use it in our [webpack config](../packages/webpack.config.js) as well in our [MDC-Web repository](https://github.com/material-components/material-components-web).
```

Comment:
This sounds overly descriptive of Autoprefixer. Maybe 
```
If using Sass instead of the pre-compiled CSS, we recommend you use [Autoprefixer](https://github.com/postcss/autoprefixer) for improved browser support. 
```
or something like that? 

Also, won't they need instructions for setting up `sass-loader` in order to compile?

Reply:
Ya good call...I'm going to cover sass loader in the getting-started.md as part of issue #67.